universal-application-tool-0.0.1/app/services/program/BlockDefinition.java

```diff
@@ -89,7 +96,7 @@ public boolean isRepeater() {
    */
   @JsonIgnore
   public boolean isRepeated() {
-    return repeaterId().isPresent();
+    return repeaterId().isPresent() && repeaterQuestionId().isPresent();
```

Comment:
Question (blocking): why is it not repeated if only `repeaterId().isPresent()`?

Reply:
I thought about this more last night and I think we can get away with not having `repeaterQuestionId` at all.